client/me/purchases/billing-history/receipt.jsx

```diff
@@ -318,7 +318,9 @@ function ReceiptLineItems( { transaction } ) {
 				<tfoot>
 					<tr>
 						<td className="billing-history__receipt-desc">
-							<strong>{ translate( 'Total' ) }:</strong>
+							<strong>
+								{ translate( 'Total paid:', { context: 'Total amount paid for product' } ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( 'Total paid', { context: 'Sum of earnings that have been distributed'} )` *ES Score: 9*


:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

Reply:
@jjchrisdiehl: interesting about the long context - can you change it to a comment instead? [We might be using `context` wrong in both instances.](https://lamda.blog/2018/03/02/psa-double-check-your-strings-for-ambiguous-contexts/)